### DISCUSSION TITLE
antvS2的表格自定义icon如何调整大小

### DISCUSSION THREAD

**Discussion by YingJian-hu** [antvis/S2#2769] [Q&A]
试了很多办法都没成功。
conditions.icon.mapping中没有对应size的属性。
setTheme({rowCell: {icon: siez: 16}})的方式也无法设置自定义icon，只能设置自带的icon

**Comment by YingJian-hu** [comment]
@lijinke666 麻烦大佬帮忙解答下，谢谢

**Comment by lijinke666** [comment]
你说的自定义 icon 是啥? customSVGIcons 还是字段标记里面的 icon, 是指动态改变字段标记的 icon 大小? 还是说通过 `customSVGIcons` 注册的 icon 没办法通过主题配置更改大小? 提供代码片段或复现 demo

**Comment by lijinke666** [comment]
https://github.com/antvis/S2/issues/1904
